### DISCUSSION TITLE
getTableActions same id

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#6888] [General]
getTableActions on a custom livewire page is giving the id of the first row in all others on an action, with the url is working fine.

same first id always
![image](https://github.com/filamentphp/filament/assets/111913304/52bfe138-69a1-4f6c-ada8-332cfc0d3bbc)

workin fine
![image](https://github.com/filamentphp/filament/assets/111913304/57e922c0-fe57-4d92-8017-9dbb2e409768)

**Comment by rmitesh** [comment]
@ramiropiro on which page you have added `getTableActions()`?
On Resource List page?

**Reply by ramiropiro** [reply]
@rmitesh ListPage same code is workin fine in a resource

**Comment by danharrin** [comment]
Try reproducing the issue in a new project and model, and you'll work out what youre doing wrong.

**Reply by ramiropiro** [reply]
@danharrin clean proyect, not working
![image](https://github.com/filamentphp/filament/assets/111913304/f5b95f4e-2db1-4da4-acf5-175e7ed2248e)
**url** method changing fine de record, but on **action** all records is from the first row, could be an issue?

**Reply by danharrin** [reply]
Please open an issue

**Reply by ramiropiro** [reply]
@danharrin partialy solved look like I had a problem in the query not sure why
![image](https://github.com/filamentphp/filament/assets/111913304/d5dcef58-0f44-4469-bb38-670c9245934e)
commented ->orWhere and now is working

**Reply by danharrin** [reply]
Wrap both the where and orWhere in a `where()` https://laravel.com/docs/10.x/queries#logical-grouping

**Reply by ramiropiro** [reply]
thanks that is a solution
